projects/swimlane/ngx-charts/ng-package.json

```diff
@@ -2,7 +2,19 @@
   "$schema": "../../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../../dist/swimlane/ngx-charts",
   "lib": {
-    "entryFile": "src/public-api.ts"
+    "entryFile": "src/public-api.ts",
+    "umdModuleIds": {
+      "d3-array": "d3-array",
+      "d3-brush": "d3-brush",
+      "d3-color": "d3-color",
+      "d3-format": "d3-format",
+      "d3-hierarchy": "d3-hierarchy",
+      "d3-interpolate": "d3-interpolate",
+      "d3-scale": "d3-scale",
+      "d3-selection": "d3-selection",
+      "d3-shape": "d3-shape",
+      "d3-time-format": "d3-time-format"
+    }
```

Comment:
Fixes:
> WARNING: No name was provided for external module 'd3-selection' in output.globals – guessing 'd3Selection'
WARNING: No name was provided for external module 'd3-brush' in output.globals – guessing 'd3Brush'
WARNING: No name was provided for external module 'd3-scale' in output.globals – guessing 'd3Scale'
WARNING: No name was provided for external module 'd3-shape' in output.globals – guessing 'd3Shape'
WARNING: No name was provided for external module 'd3-array' in output.globals – guessing 'd3Array'
WARNING: No name was provided for external module 'd3-interpolate' in output.globals – guessing 'd3Interpolate'
WARNING: No name was provided for external module 'd3-format' in output.globals – guessing 'd3Format'
WARNING: No name was provided for external module 'd3-color' in output.globals – guessing 'd3_color'
WARNING: No name was provided for external module 'd3-hierarchy' in output.globals – guessing 'd3Hierarchy'
WARNING: No name was provided for external module 'd3-time-format' in output.globals – guessing 'd3TimeFormat'